### ISSUE TITLE
Add some extra explanation on Open Canvas quadrants

### ISSUE THREAD

**Issue by aschrijver** [mozilla/open-leadership-training-series#95]
Hi @acabunoc 

Filling in the Open Canvas for OL7. Nice work!

I have a slight improvement you can consider, and that is to address each quadrant with a paragraph of explanatory text, adding additional rationale to having the quadrant, in order to help people interpret them correctly.

For instance:  

- Problem quadrant: Do I only mention product-/community-related external problems (probably), or also internal ones. E.g. external: _"Most people are unaware of harms of technology and how they affect them"_ and internal: _"Though project members express worry about tech harms, it is very hard to activate them to work on their solutions"_.

- Key metrics: Should metrics be quantitative, qualitative or even subjective (e.g. _"Frequent mention of our community in authoritative newspapers"_)

Just a tip. The paragraphs can be very short.

**Comment by abbycabs** [comment]
Huge thanks for this Arnold! I tend to leave things more open to interpretation, so people can answer in whatever way they think is useful. But I realize how that can lead to more confusion! Do you have a suggestion on to refine the current explainer text in the canvas?

I appreciate your insight here!

**Comment by aschrijver** [comment]
> I tend to leave things more open to interpretation

Yes, that is not bad in itself, as there are multiple interpretations. I think the canvas should be used in ways that are most practical and may be different for each individual case.

Some general points:

- After the first sentence in MVP paragraph, add that you best fill in the canvas with the scope of that MVP in mind. It is easy to go into too much detail or too general and the limited space of the canvas plus MVP in mind, helps scrapping that and only mention bare essentials.

- Also I think the canvas is more of an internal tool, to focus the team itself and get them aligned to set the right priorities. When you fill in for an external audience one tends to fill in more 'beautiful' well-formed sentences, but distract from the essence of the canvas. You could mention the distinguisment.

- Also mentioning practicality and freedom to make changes to applying the canvas, can be good. At [Humane Tech Community](https://community.humanetech.com) our eventual scope, audience and stakeholders are vast. The canvas and MVP concept fit best for a concrete project or product, while our scope is more fitting to creating a 'product portfolio'. We created a single top-level canvas, and will probably later create lower-level ones, i.e. targeted to our focus areas of Wellbeing, Freedom and Society (and maybe one for Harms), etc.

The hardest to fill in - and mostly likely to lead to confusion - are **Solution**, **Problem**, **Unique Value Proposition**. I mentioned a point about Problem above, for the others:

- Solution quadrant: Do I describe them conceptually or concreetly (e.g. "We collect solutions for the harms of technology, which we will promote to the public to stimulate their use" versus "We collect technical solutions on Github, discuss them on our forum, after which our Campaigners promote them via our Awareness Program"). Both approaches can be valid, I think, but maybe you should stick with the approach you've chosen.

- UVP quadrant: "What you offer" and "Why you are different". The first has overlap with Solution. The second statement is more important, I think. It is what makes you stick out above other similar initiatives. Your unique position that'll make your project viable. Note that you are missing that last aspect in your second example.

The other quadrants are more or less clear, only in our case as a community we read 'users' as our 'audience' and 'contributors' as our 'members' divided into several groups (plus partners).

**Comment by aschrijver** [comment]
FYI, this is how we filled out our Open Canvas for OL7 Cohort B: https://github.com/MozillaFestival/open-leaders-7/issues/41#issuecomment-462317076
